community/artifactory-ha/values.yaml

```diff
@@ -235,9 +395,18 @@ artifactory:
       path: "artifactory-ha/filestore"
       refreshCredentials: true
       testConnection: false
+      s3AwsVersion: "AWS4-HMAC-SHA256"
+
       ## Additional properties to set on the s3 provider
       properties: {}
       #  httpclient.max-connections: 100
+    ## For artifactory.persistence.type azure-blob
+    azureBlob:
+      accountName:
+      accountKey:
+      endpoint:
+      containerName:
+      testConnection: false
   service:
     name: artifactory
```

Comment:
I don't see that `.Values.artifactory.service.name` is used in any templates; it should be removed.